feelpp/feel/feeldiscr/quadptlocalization.hpp

```diff
@@ -1269,7 +1262,7 @@ precompute(vf::detail::LinearForm<FE,VectorType,ElemContType> const& __form)
     typedef typename linearformContext<FE,VectorType,ElemContType>::gmc_form_ptrtype gmc_form_ptrtype;
     typedef typename linearformContext<FE,VectorType,ElemContType>::pc_form_type pc_form_type;
     typedef typename linearformContext<FE,VectorType,ElemContType>::pc_form_ptrtype pc_form_ptrtype;
-
+    static const size_type gmc_v = linearformContext<FE,VectorType,ElemContType>::gmc_v;
```

Comment:
use `inline`
```
static inline const size_type ...
```
`constexpr` would work as well 

see eg https://www.codingame.com/playgrounds/2205/7-features-of-c17-that-will-simplify-your-code/inline-variables